requirements.txt

```diff
@@ -8,6 +8,7 @@ PyYAML==3.12
 requests==2.13.0
 SQLAlchemy==1.1.9
 suds-jurko==0.6
+yaml
```

Comment:
Please pin the version here.

On ``requirements.txt`` goes an specific version that you ensure that works. On ``setup.py`` goes the minimal one that should work.